RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h

```diff
@@ -19,25 +27,20 @@ class HcalDeterministicFit {
   HcalDeterministicFit();
   ~HcalDeterministicFit();
 
-  void init(HcalTimeSlew::ParaSource tsParam, HcalTimeSlew::BiasSetting bias, NegStrategy nStrat, PedestalSub pedSubFxn_);
+  void init(HcalTimeSlew::ParaSource tsParam, HcalTimeSlew::BiasSetting bias, NegStrategy nStrat, PedestalSub pedSubFxn_, double parhb0, double parhb1, double parbe0, double parbe1, double parhe0, double parhe1);
```

Comment:
Are these new parameters more appropriately a part of one of the available parameters already?
If do, please add them there.
If not, it would still be better to put them into a struct and pass around as one argument